### PR TITLE
Update docs with ViewCommand information

### LINKED ISSUE #139: Update documentation with ViewCommand

### PR DESCRIPTION
**Pull request by Andrew22Teoh**
This update adds:
1. User guide feature description of the view command, with an attached photo of the view window
2. Developer guide information on the view command, including use cases
3. Minor formatting changes on Use Cases headers for better readability
4. Updated UI class diagram

Closes #139

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -376,7 +376,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-**Use case: Delete a contact**
+##### **Use case: View all information on a contact**
```
Comment (AbdulrahmanAlRammah) [inline docs/DeveloperGuide.md]: Should be "of" a contact not "on" a contact.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ SocialBook is a **desktop app for managing contacts, optimized for use via a  Li
 
    * `list` : Lists all contacts.
 
+   * `view 2` : Views all the information on the 2nd contact shown in the current list.
```
Comment (AbdulrahmanAlRammah) [inline docs/UserGuide.md]: Change to: Displays all the information of the 2nd contact shown in the current list.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -105,6 +107,20 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
+### Viewing a person : `view`
+
+Pops up a window containing all the information that has been saved on this person. Also filters the current listed persons to just this specific person.
+
+Format: `view INDEX`
+
+* This command only works to `view` a single person. Trying to key in more than 1 index will result in an error message.
```
Comment (AbdulrahmanAlRammah) [inline docs/UserGuide.md]: Change first sentence to: This command only allows the viewing of one person at a time.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -105,6 +107,20 @@ Shows a list of all persons in the address book.
...
+* Viewing is done by index, and **not** the person's name or any other field. Attempting to `view` by name, address, or any other fields will result in an error.
+
+Examples:
+* `view 1` will bring up the help window for the first person in the displayed list. <br>
```
Comment (AbdulrahmanAlRammah) [inline docs/UserGuide.md]: HELP is a mistake. It should say "view window".

**Review by AbdulrahmanAlRammah — CHANGES_REQUESTED** [review]
See Minor phrasing changes. Also there are no changes to the UI class diagram. There are no changed files relating to a class diagram.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Andrew22Teoh) [reply docs/DeveloperGuide.md]: Changed to "about a contact"

File: docs/UserGuide.md (hunk 4 above)
Comment (Andrew22Teoh) [reply docs/UserGuide.md]: Good catch, thank you!

**Review by AbdulrahmanAlRammah — APPROVED** [review]
LGTM

**Pull request merged**